tests/tests_twobody/test_propagation.py

```diff
@@ -463,3 +463,22 @@ def test_propagator_with_zero_eccentricity(propagator):
     assert_quantity_allclose(orbit.inc, res.inc)
     assert_quantity_allclose(orbit.raan, res.raan)
     assert_quantity_allclose(orbit.argp, res.argp)
+
+
+@pytest.mark.parametrize("propagator", ALL_PROPAGATORS)
+def test_after_propagation_r_and_v_dimensions(propagator):
```

Comment:
This is not required, there is a test already checking for `ALL_PROPAGATORS`, see:

https://github.com/poliastro/poliastro/blob/0a0c00559a36ce9779f824738773bc7e98a6697c/tests/tests_twobody/test_propagation.py#L105-L106

Reply:
The motivation for adding this test was this line: https://github.com/poliastro/poliastro/blob/main/src/poliastro/twobody/propagation.py#L476. I noticed that below the "TODO"'s, there was an assert condition inside the source code. I though it would be good to turn it into its own test.  Also, this method uses the propagator method directly and not the propagation method of an orbit. Hence, after adding this test, the assert condition in `src` was removed.

What do you think about adding this test?

Reply:
I'm +1 on adding this test and removing the `assert`!

Reply:
Oh, I see. Thanks for the clarification @Yash-10!